src/Language/PureScript/Sugar/Names/Exports.hs

```diff
@@ -19,7 +19,7 @@ import Language.PureScript.AST
 import Language.PureScript.Crash
 import Language.PureScript.Errors
 import Language.PureScript.Names
-import Language.PureScript.Sugar.Names.Env

```

Comment:
All the lines that have changed but don't look different in this file are line-ending changes.